src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -199,7 +199,7 @@ def get_api_lvs():
           ;/dev/ubuntubox-vg/swap_1;swap_1;ubuntubox-vg
 
     """
-    fields = 'lv_tags,lv_path,lv_name,vg_name,lv_uuid'
+    fields = 'lv_tags,lv_path,lv_name,vg_name,lv_uuid,devices'
```

Comment:
This is going to cause issues for systems where logical volumes have been created from parts of different underlying pvs. For example, this `sliced/lv` is using /dev/sdi and /dev/sdh, will report the same lv and same vg:

    root@node9:/home/vagrant# lvs sliced/lv -o +devices
      LV   VG     Attr       LSize   Pool Origin Data%  Meta%  Move Log Cpy%Sync Convert Devices
      lv   sliced -wi-a----- 400.00m                                                     /dev/sdi(50)
      lv   sliced -wi-a----- 400.00m                                                     /dev/sdh(0)

This is problematic because it will break uniqueness on reporting. This is how the actual API call in ceph-volume would see in this case for all lvs (note how 'sliced' appears twice)

     lvs --noheadings --readonly --separator=";" -o lv_tags,lv_path,lv_name,vg_name,lv_uuid,devices
      ;/dev/ceph/test;test;ceph;Ea7Kj7-HSKe-EFrp-Ht8A-HRQV-zPMt-ktExtq;/dev/sdd(0)
      ;/dev/sliced/lv;lv;sliced;5vVQ2s-acKA-9vCP-Dafo-9c8e-d7gB-HtcTdb;/dev/sdi(50)
      ;/dev/sliced/lv;lv;sliced;5vVQ2s-acKA-9vCP-Dafo-9c8e-d7gB-HtcTdb;/dev/sdh(0)
      ;/dev/ubuntubox-vg/root;root;ubuntubox-vg;Z2gyTg-bQau-GvZb-Uqty-WHur-xcP3-v2usYb;/dev/sda5(0)
      ;/dev/ubuntubox-vg/swap_1;swap_1;ubuntubox-vg;fbzgJ6-BqEm-Rb99-ou3N-gqek-liGR-p2ZI6h;/dev/sda5(76163)

Reply:
This is why I've sent you the email asking about weird lvm setups like this, because it makes pvs reporting more complicated than the "one-liner" solution I have on this PR. I will update the pr on monday and remove this part.